src/Extension/SiteTreeExtension.php

```diff
@@ -1,8 +1,16 @@
 <?php
-class RealMeSiteTreeExtension extends DataExtension
+
+namespace SilverStripe\RealMe\Extension;
+
+use SilverStripe\RealMe\RealMeService;
+use SilverStripe\Security\Member;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\DataExtension;
+
+class SiteTreeExtension extends DataExtension
 {
     private static $dependencies = array(
-        'service' => '%$RealMeService'
+        'service' => '%$' . RealMeService::class
```

Comment:
By default extensions in SilverStripe 4 are loaded as singletons, which means that this service will also be loaded as a singleton - is that OK? I suspect it probably is

Reply:
Yeah it doesn't matter. Services shouldn't carry state, so it _shouldn't_ make a difference.